test/views/movement/MovementConfirmationDepartureViewSpec.scala

```diff
@@ -41,13 +41,24 @@ class MovementConfirmationDepartureViewSpec extends ViewSpec {
       )
     }
 
-    "have back to starttest button" in {
+    "have back to start button" in {
 
       val backButton = page(consignmentReferences).getElementsByClass("button").first()
 
       backButton must containMessage("site.backToStart")
       backButton must haveHref(controllers.routes.ChoiceController.displayPage())
     }
+
+    "have 'view requests' link" in {
+      val statusInfo = page(consignmentReferences).getElementById("status-info")
+      statusInfo.getElementsByTag("a").get(0) must haveHref(controllers.routes.ChoiceController.startSpecificJourney(forms.Choice.ViewSubmissions))
+    }
+
+    "have 'next steps' link" in {
+      val nextSteps = page(consignmentReferences).getElementById("next-steps")
+      nextSteps.getElementsByTag("a").get(0) must haveHref(controllers.routes.ChoiceController.startSpecificJourney(forms.Choice.Departure))
```

Comment:
I'll test this in different way, not getting elements by tags, but try by ID, but not sure that we have id or if we have helpers method to check it in different way.